Source/AMRInterpolator/AMRInterpolator.hpp

```diff
@@ -79,6 +95,16 @@ template <typename InterpAlgo> class AMRInterpolator
     // A bit of Android-ism here, but it's really useful!
     // Identifies the printout as originating from this class.
     const static string TAG;
+
+    // Variables for symmetric BC
+    // m_bc_params can't be a 'const' reference as we need a
+    // constructor with backward compatibility that builds an artificial
+    // 'BoundaryConditions::params_t'
+    BoundaryConditions::params_t m_bc_params;
+    /// simplified bools saying whether or not boundary has
+    /// a reflective condition in a given direction
+    std::array<bool, CH_SPACEDIM> m_lo_boundary, m_hi_boundary;
```

Comment:
I think these should have a slightly more descriptive name e.g. `m_lo_boundary_symmetric` or `m_lo_boundary_reflective`

Reply:
Done